initializr/src/main/groovy/io/spring/initializr/web/MainController.groovy

```diff
@@ -125,6 +126,12 @@ class MainController extends AbstractInitializrController {
 		renderHome('home.html')
 	}
 
+	@RequestMapping(value = '/dependencies.json', produces = 'application/json')
```

Comment:
I've added a new endpoint here. It's needed by the autocompletion engine.
We'll probably need to add more tests for this.